web/pdf_viewer.js

```diff
@@ -383,8 +383,8 @@ var PDFViewer = (function pdfViewer() {
             return;
           }
           var getPagesLeft = pagesCount;
-          for (var pageNum = 1; pageNum <= pagesCount; ++pageNum) {
-            pdfDocument.getPage(pageNum).then(function(pageNum, pdfPage) {
+          for (let pageNum = 1; pageNum <= pagesCount; ++pageNum) {
```

Comment:
this was just converted into "var" for translated code, do we need to move let inside for-block?